vhdbuilder/packer/generate-vhd-publishing-info.sh

```diff
@@ -11,6 +11,29 @@ required_env_vars=(
     "IMAGE_VERSION"
 )
 
+# Higher the replication_inverse, lower is the usage and number of replicas
+set -x
+REPLICATION_INVERSE=1
+feature_set=("fips" "gpu" "arm64" "cvm" "tl" "1804")
+if [ "${OFFER_NAME,,}" != "ubuntu" ]; then
+    # Since Ubuntu is our most used SKU as compared to Windows/Mariner/AzLinux, we dont need the same number of replicas for all.
+    # Starting off with half replicas.
+    REPLICATION_INVERSE=$((REPLICATION_INVERSE * 2))
```

Comment:
yeah, will need confirmation from @AbelHu on this

Reply:
yep, will take this PR to Henry, Charlie, and Abel

Reply:
got approval from Henry and Charlie, this PR just adds to Windows publishing info but doesnt consume it, Windows team can pick it if they want to alter replicas.